spec/development/workload/PageIterator.ts

```diff
@@ -0,0 +1,56 @@
+import { PageIterator, PageIteratorCallback } from "../../../src/tasks/PageIterator";
+import { Event } from "microsoft-graph";
+import { assert } from "chai";
+import { getClient } from "../test-helper";
+
+const client = getClient();
+describe("PageIterator", function() {
+	describe("sameHeadersPassedWithPageIterator", () => {
+		it("verify same headers", async () => {
+			const response = await client
+				.api(`/me/events?$top=2`)
+				.headers({ Prefer: 'outlook.timezone= "pacific standard time"' })
```

Comment:
constant

Reply:
Curious to know what you mean with `constant`

Reply:
both the key (outlook.timezone) and the value (pacific standard time) should be in a constant instead of being repeated IMHO. It makes ref counting/refactoring easier and avoids typos etc.

Reply:
Used constants where possible.